src/App.tsx

```diff
@@ -1,14 +1,70 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { FilterCriteria, Select, Status } from './types/otherTypes';
+
+const filterTodos = (todos: Todo[], filter: FilterCriteria): Todo[] => {
+  const { status, searchInput } = filter;
+
+  return todos.filter(todo => {
+    if (status === 'active' && todo.completed) {
+      return false;
+    }
+
+    if (status === 'completed' && !todo.completed) {
+      return false;
+    }
+
+    if (
+      searchInput &&
+      !todo.title.toLowerCase().includes(searchInput.toLowerCase())
+    ) {
+      return false;
+    }
+
+    return true;
+  });
+};
 
 export const App: React.FC = () => {
+  const [loadingTodos, setLoadingTodos] = useState(true);
```

Comment:
```suggestion
  const [isTodosLoading, setIsTodosLoading] = useState(true);
```
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig